C-Plus-Plus/cp/Three_Sum.cpp

```diff
@@ -0,0 +1,70 @@
+/*
+Description :
+    
+	Three Sum Problem - In this problem, user will provide an array and we
+	                    have find wether there is any "three" numbers are 
+						present in the array who's sum is equal to the target 
+						provided.If it is present then it will show true else 
+						it will show false.
+
+*/
+
+
+#include<bits/stdc++.h>
+using namespace std;
+
+int main()
+{
+	int n; // n= size of an array
+	cin>>n;
+	int target; //target = the number who's sum we have find.
+	cin>>target;
+
+	vector<int> a(n); //n size of array
+
+	for(auto &i : a){ //taking input in the array
+		cin>>i;
+	}
+	bool found = false;
+	sort(a.begin(),a.end());
+
+	for(int i=0;i<n;i++){   
+		int lo =i+1, hi = n-1;
+		while(lo<hi){
+			int current = a[i]+a[lo]+a[hi];
+			if(current == target){
+				found = true;
+			}
+			if(current<target){
+				lo++;
+			}
+			else{
+				hi--;
+			}
+		}
+	}
+	if(found){
+		cout<<"True";
```

Comment:
Display a message in Print form, as what are you showing output to the user. Ex: What is True?

Reply:
I have written in the description what does true/false means .

Reply:
User, will not going to see your description message. It's only for easily understandable format. User will only experience the User-interface, which only shows True or False that doesn't **depicts any value to the understanding to the user**.
**Better to work on the suggestion as suggested earlier**.
Hope it make sense.

Reply:
So should I change the cout message and change it to more understandable ?

Reply:
Yes, add some more info regarding that